--- signon-8.56/src/remotepluginprocess/remotepluginprocess.pro.link	2015-03-26 17:36:45.854835523 +0000
+++ signon-8.56/src/remotepluginprocess/remotepluginprocess.pro	2015-03-26 17:37:51.064874334 +0000
@@ -32,9 +32,10 @@ system(pkg-config --exists libproxy-1.0)
 }
 
 QMAKE_LIBDIR += \
+    $${TOP_BUILD_DIR}/lib/plugins \
     $${TOP_BUILD_DIR}/lib/plugins/signon-plugins-common
 
-LIBS += -lsignon-plugins-common
+LIBS += -lsignon-plugins -lsignon-plugins-common
 
 QMAKE_CXXFLAGS += -fno-exceptions \
                   -fno-rtti
